kafka/kafka_source.go

```diff
@@ -84,7 +85,7 @@ func (k *KafkaSource) Generate(out chan<- interface{}) {
 	if kconf.SASLEnabled {
     		//sarama config plain by default
     		config.Net.SASL.User = kconf.SASLUsername
-    		config.Net.SASL.Password = os.Getenv("KAFKA_CLIENT_USER_PASSWORD")
+    		config.Net.SASL.Password = os.Getenv(kconf.SASLPasswordKey)
```

Comment:
is this change needed?

Reply:
Created this branch from main. Develop branch and main branch differ here